client/landing/stepper/index.tsx

```diff
@@ -44,7 +44,15 @@ window.AppBoot = async () => {
 
 	const queryClient = new QueryClient();
 
-	const flow = config.isEnabled( 'signup/anchor-fm' ) ? anchorFmFlow : siteSetupFlow;
+	const sanitizePodcastId = ( id: string | null ) => id?.replace( /[^a-zA-Z0-9]/g, '' );
+	const search = location.search;
+
+	const anchorPodcastId = sanitizePodcastId(
+		new URLSearchParams( search ).get( 'anchor_podcast' )
+	);
+
+	const flow =
+		anchorPodcastId && config.isEnabled( 'signup/anchor-fm' ) ? anchorFmFlow : siteSetupFlow;
```

Comment:
My inclination is to say this could be updated to be a little more easily readable/obviously extensible - something like:

```
let flow = siteSetupFlow;
if (anchorPodcastId && config.isEnabled( 'signup/anchor-fm' )) {
    flow = anchorFmFlow;
}
```

Otherwise this looks good though.